src/index.ts

```diff
@@ -378,6 +392,49 @@ export function createMercuriusTestClient(
     })
   }
 
+  const getFederatedEntity = async ({
+    typename,
+    keys,
+    typeQuery,
+  }: {
+    typename: string
+    keys: Record<string, string | number>
+    typeQuery: string
+  }) => {
+    let stringifiedKeys: string[] = []
+
+    for (const key in keys) {
+      const value = typeof keys[key] === 'number' ? keys[key] : `"${keys[key]}"`
+      stringifiedKeys.push(`${key}: ${value}`)
+    }
+
+    try {
+      const result = await query(`
+      query {
+          _entities(representations: [{ __typename: "${typename}", ${stringifiedKeys.join(
```

Comment:
I would prefer we used variables instead.

Reply:
🤦 Thanks for suggestion.

Reply:
Done, please take a look.